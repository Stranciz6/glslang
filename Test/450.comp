#version 450 core
layout(local_size_x = 0) in; // ERROR, 0 not allowed