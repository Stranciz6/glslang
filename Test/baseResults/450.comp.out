450.comp
ERROR: 0:2: 'local_size_x' : must be at least 1 
ERROR: 1 compilation errors.  No code generated.


Shader version: 450
local_size = (1, 1, 1)
ERROR: node is still EOpNull!
0:?   Linker Objects


Linked compute stage:

ERROR: Linking compute stage: Missing entry point: Each stage requires one entry point

Shader version: 450
local_size = (1, 1, 1)
ERROR: node is still EOpNull!
0:?   Linker Objects

